### ISSUE TITLE
Python 2 Support

### ISSUE THREAD

**Issue by mas-dse-juremigi** [lipoja/URLExtract#8]
Is there a python 2 version of this? Would love to use this for a python 2 project I am working on.

Also, are there unit tests? It would be great to see what test cases are used validation.

similar to this:
https://mathiasbynens.be/demo/url-regex

thanks, keep up the great work!

**Comment by lipoja** [comment]
Hi,
I saw @penafieljlm porting it to 2.7 but I did not use hi version. Maybe he will tell you more about the port. https://github.com/penafieljlm/URLExtract-py2.7

I will make similar tests as you suggested, the table looks good. 
To answer your question: I have only few doc tests. I have to create better test cases and unit tests. This is really the thing that I am missing most. (except of time)

Thank you for your comment. 
I am going to do the unit test when I have little bit of free time.
